users/forms/profile_form.py

```diff
@@ -0,0 +1,27 @@
+from django import forms
+from django.contrib.auth.models import User
+
+
+class ProfileForm(forms.ModelForm):
+    date_of_birth = forms.DateField(widget=forms.SelectDateWidget())
+    bio = forms.CharField(widget=forms.Textarea, required=False)
+
+    class Meta:
+        model = User
+        fields = [
+            'first_name',
+            'last_name',
+            'email',
+            'date_of_birth',
+            'bio',
+        ]
+
+
+    def save(self, request, commit=True):
```

Comment:
@ahmadx16 Why you are setting properties one by one, I think modelForm should take care of that.